### ISSUE TITLE
LAGUNA: LowerCentralSeries stops with a central group

### ISSUE THREAD

**Issue by hungaborhorvath** [gap-system/gap#3282]
````
gap> g:=DihedralGroup(8);;
gap> f:=GF(4);;
gap> fg:=GroupRing(f,g);;
gap> e:=Identity(fg);;
gap> u:=Units(fg);;
gap> LowerCentralSeriesOfGroup(u);
[ <group of size 49152 with 7 generators>, 
  <group of size 192 with 21 generators>, <group of size 3 with 1 generators> 
 ]
gap> C := LowerCentralSeriesOfGroup(u)[3];;
gap> IsCentral(u, C);
true
````

This is clearly wrong, as the group is nilpotent, and Lower Central Series should end with the trivial group. Apparently, gap knows that the last term is central, so must be something wrong with the generic method of the Lower Central Series calculation. 

I will take a look at this later. 

Edit: GAP banner
````
 ┌───────┐   GAP 4.10.0 of 01-Nov-2018
 │  GAP  │   https://www.gap-system.org
 └───────┘   Architecture: x86_64-pc-linux-gnu-default64
 Configuration:  gmp 6.1.2, readline
 Loading the library and packages ...
 Packages:   AClib 1.3.1, Alnuth 3.1.0, AtlasRep 1.5.1, AutoDoc 2018.09.20, 
             AutPGrp 1.10, Browse 1.8.8, Carat 2.2.2, CRISP 1.4.4, 
             Cryst 4.1.18, CrystCat 1.1.8, CTblLib 1.2.2, FactInt 1.6.2, 
             FGA 1.4.0, GAPDoc 1.6.2, IO 4.5.4, IRREDSOL 1.4, LAGUNA 3.9.0, 
             Polenta 1.3.8, Polycyclic 2.14, PrimGrp 3.3.2, RadiRoot 2.8, 
             ResClasses 4.7.1, SmallGrp 1.3, Sophus 1.24, SpinSym 1.5, 
             TomLib 1.2.7, TransGrp 2.0.4, utils 0.59
 Try '??help' for help. See also '?copyright', '?cite' and '?authors'
````

**Comment by hungaborhorvath** [comment]
Ok, the problem seems to be that gap calculates wrongly the inverse of one of the generators, and therefore commutator calculation is messed up. 
````
gap> GeneratorsOfGroup(u);
[ (Z(2)^0)*f1*f2*f3, (Z(2^2))*f1*f2*f3, (Z(2)^0)*f2*f3, 
  (Z(2)^0)*f2*f3+(Z(2^2))*f1*f2*f3, (Z(2)^0)*f1*f3+(Z(2^2))*f1*f2*f3, 
  (Z(2)^0)*f1*f3+(Z(2)^0)*f2*f3+(Z(2)^0)*f1*f2*f3, 
  (Z(2)^0)*f1*f3+(Z(2)^0)*f2*f3+(Z(2^2))*f1*f2*f3 ]
gap> Order(u.4);
12
gap> u.4^11=u.4^(-1);
false
gap> for n in [1..12] do if u.4^n=u.4^(-1) then Print(n); fi; od; 
7
gap> u.4*C.1*u.4^(-1)*C.1^(-1);
(Z(2^2))*<identity> of ...
gap> u.4*C.1*u.4^11*C.1^(-1);
(Z(2)^0)*<identity> of ...
````

**Comment by fingolfin** [comment]
I checked and this issue also appears in GAP 4.9 and 4.8

**Comment by fingolfin** [comment]
The problem goes away if I start with `gap -A`, and in the end, seems to be in the LAGUNA package (ping @alex-konovalov ). It is fixed by this patch:
```patch
diff --git a/lib/laguna.gi b/lib/laguna.gi
index 25c657a..7b7a2c0 100644
--- a/lib/laguna.gi
+++ b/lib/laguna.gi
@@ -465,7 +465,7 @@ InstallOtherMethod( InverseOp,
             pow := pow * (-x);
           od;
 
-          return a*inv;
+          return a^-1*inv;
         fi; 
       else        
         TryNextMethod(); # since our case is not modular                  
```

**Comment by ChrisJefferson** [comment]
Good find!

**Comment by fingolfin** [comment]
@alex-konovalov fix is at https://github.com/gap-packages/laguna/pull/13

**Comment by fingolfin** [comment]
This is fixed in GAP 4.10.1
